Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: build start test lint compare shell
+
+build:
+	docker build --tag ensemble-transposer .
+
+start: build
```

Comment:
Maybe making `build` a dependency here is a bad idea. If I always want the latest versions of all files to be available in Docker, would it be better to use a volume?

Reply:
For local development I would say volumes make it easier and more transparent as you're editing files. Docker compose has a nice syntax for it, but you can mount volumes with straight docker as well.

Reply:
On second thought, I think I may stick with not using volumes for now to keep things simple, since the build process is usually very quick, but I could be talked out of that.